src/components/MovieListPage/movieListPage.jsx

```diff
@@ -80,15 +112,10 @@ const MovieListPage = (props) => {
 
     return (
         <>
-            {showDetail && selectedMovie ? <MovieDetails movie={selectedMovie}/> : 
-            <>
-                <div className='add-movie-div'>
-                    <button className='add-movie-button' onClick={handleAddClick}>+ Add Movie</button>
-                </div>
-                <div className='movie-list-page'>
-                    <SearchForm initialSearchQuery={searchQuery} onSearch={handleSearchSubmit} />
-                </div>
-            </>}
+            <div className='add-movie-div'>
+                <button className='add-movie-button' onClick={handleAddClick}>+ Add Movie</button>
+            </div>
+            {movieId ? <Outlet /> : <SearchForm initialSearchQuery={searchQuery} onSearch={handleSearchSubmit} setSearchQuery={setSearchQuery}/> }
```

Comment:
wrong props names of the components + no setSearchQuery prop inside the component

Reply:
Done.